### PR TITLE
Portfolio component

### LINKED ISSUE #27: Portfolio Component

### PR DESCRIPTION
**Pull request by Archiving**
Closes #27

### CODE REVIEW

File: client/src/components/Portfolio.tsx [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+        fontWeight: "bold",
+        marginLeft: theme.spacing(20)
+    },
+    changeValueComponent: {
+        color: "#00FF00"
+    }
+}))
+
+function numberWithCommas(valueStr: string) {
+    return valueStr.replace(/\B(?=(\d{3})+(?!\d))/g, ",");
```
Comment (JohnTurkson) [inline client/src/components/Portfolio.tsx:30]: We should swap this out later on, but not a priority for now.

File: client/src/components/Portfolio.tsx [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+    })
+
+    return value
+}
+
+const Portfolio = (props: any) => {
+
+    const classes = useStyles()
+    return (
+        <div className={classes.container}>
```
Comment (JohnTurkson) [inline client/src/components/Portfolio.tsx:47]: Is it possible to replace this with `<Container component="div">...`? I'm not sure if it's recommended to mix material UI items with regular HTML.

Comment (JohnTurkson) [reply client/src/components/Portfolio.tsx:47]: Also for `<div>`s below.

File: client/src/components/Portfolio.tsx [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+    changeValueComponent: {
+        color: "#00FF00"
+    }
+}))
+
+function numberWithCommas(valueStr: string) {
+    return valueStr.replace(/\B(?=(\d{3})+(?!\d))/g, ",");
+}
+
+function calculatePortfolioValue(cryptos: any[]): number {
```
Comment (JohnTurkson) [inline client/src/components/Portfolio.tsx:33]: Might want to add a TODO for the `any[]` here, so we can find it later.

File: client/src/containers/CryptoPriceList.tsx [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+    display: flex;
+    flex-direction: row;
+`
+
+// TODO: change any type
+const createCryptoListing = (cryptos: any[]): JSX.Element => {
+    // return Coin components based on JSON data
+    return(
+
+        <div>{cryptos.map(
```
Comment (JohnTurkson) [inline client/src/containers/CryptoPriceList.tsx:16]: Replace with `<Container component="div">` if possible.

File: client/package.json [hunk 5]
```diff
@@ -44,5 +44,9 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
+  },
+  "devDependencies": {
+    "@types/react-router-dom": "^5.1.7",
+    "@types/styled-components": "^5.1.10"
```
Comment (JohnTurkson) [inline client/package.json:50]: Good call. We're not exporting any packages, so the `@types` packages should be in here. We should update the any others as necessary in the future.

**Review by JohnTurkson — CHANGES_REQUESTED** [review]
Overall LGTM, just small changes

File: client/src/components/Portfolio.tsx (hunk 1 above)
Comment (JohnTurkson) [reply client/src/components/Portfolio.tsx:30]: Might want to add a TODO for above.

File: client/src/components/Portfolio.tsx (hunk 3 above)
Comment (Archiving) [reply client/src/components/Portfolio.tsx:33]: used <Box> for flex box features that are needed.

**Pull request merged**